src/Models/Subscription.php

```diff
@@ -22,6 +22,8 @@
  */
 class Subscription extends PhalconSubscription
 {
+
+    const DEFAULT_GRACE_PERIOD_DAYS = 5;
```

Comment:
@rwhite27 do you think we also need to have this variable on the app settings ?

Reply:
Thought about it but we really can't change grace period days on stripe per app. Stripe won't let us change via API.